addons/l10n_de_stock/models/stock.py

```diff
@@ -3,12 +3,16 @@
 
 
 class StockInventory(models.Model):
-    _inherit = 'stock.inventory'
+    _inherit = 'stock.picking'
```

Comment:
heu? but the class stays StockInventory ?

Reply:
Not anymore

Reply:
better